app/assets/javascripts/map/components/registration-form.js

```diff
@@ -11,6 +14,8 @@ function RegistrationForm() {
     startingAt: null,
   }
 
+  let validateEmail = () => regexForEmailValidation.test(data.email) ? alert = null : alert = { type: 'error', message: Util.translate('registration.form.email_invalid')};
```

Comment:
I think I made a mistake in my previous review comment, it looks like the code expects `status: 'error'` rather than `type: 'error'`

If you do that it will colour the alert correctly automatically.